### PR TITLE
j

### PR DESCRIPTION
**Pull request by jrobledob**

### CODE REVIEW

File: file-to-edit.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-write your name below in bullets. This name must eventually reflect in a main branch of the original repository.
+write your name below in bullets. This name must eventually be reflected in a main branch of the original repository.
 
 1. Krishna
-2.
+2. Jacobo Robledo
```
Comment (krishnakeshav) [inline file-to-edit.txt:4]: This is not quite right. Please change

**Review by krishnakeshav — APPROVED** [review]
looks good

File: file-to-edit.txt (hunk 1 above)
Comment (jrobledob) [reply file-to-edit.txt:4]: Sorry that is my name.

**Review by jrobledob — COMMENTED** [review]
sorry

**Review by jrobledob — COMMENTED** [review]
changed

**Pull request merged**